package.json

```diff
@@ -17,12 +17,13 @@
   "homepage": "https://github.com/firstandthird/eslint-config-firstandthird#readme",
   "devDependencies": {
     "chai": "^3.5.0",
-    "eslint": "^4.13.0",
-    "eslint-plugin-import": "^2.8.0",
+    "eslint": "^6.7.2",
     "mocha": "^3.1.2"
   },
   "peerDependencies": {
-    "eslint": ">=4.0.0",
-    "eslint-plugin-import": "^2.18.0"
+    "eslint": ">=4.0.0"
+  },
+  "dependencies": {
+    "eslint-plugin-import": "^2.19.1"
```

Comment:
does this work as a dep now?  before we had to include explicitly.  Maybe we should have an example folder so we can make sure things are still working